docker/prover/Dockerfile

```diff
@@ -21,9 +19,23 @@ RUN curl https://sh.rustup.rs -sSf | bash -s -- -y && \
 WORKDIR /usr/src/zksync
 COPY . .
 
-ENV BELLMAN_CUDA_DIR=/usr/src/zksync/bellman-cuda
-
-RUN CARGO_HOME=./cargo cargo build --release
+RUN set -e; \
+    if [ "$ERA_BELLMAN_CUDA_RELEASE" = "latest" ]; then \
+    api_url="https://api.github.com/repos/matter-labs/era-bellman-cuda/releases/latest"; \
+    else \
+    api_url="https://api.github.com/repos/matter-labs/era-bellman-cuda/releases/tags/${ERA_BELLMAN_CUDA_RELEASE}"; \
+    fi; \
+    release_info=$(curl --silent "$api_url" | jq -r ".tarball_url, .assets[0].browser_download_url"); \
```

Comment:
FYI: This will be rate limited to 60 per hour.

Reply:
but we don't have other choice here, right?

Reply:
changed it to using API only on fallback to latest release/prerelease